src/dialogs.c

```diff
@@ -53,17 +64,28 @@ void oregano_error_with_title (gchar *title, gchar *desc)
 		span_msg = g_string_append (span_msg, desc);
 	}
 
-	dialog = gtk_message_dialog_new_with_markup (NULL, GTK_DIALOG_MODAL, GTK_MESSAGE_ERROR,
-	                                             GTK_BUTTONS_OK, "%s", span_msg->str);
+	dialog = gtk_message_dialog_new (NULL, GTK_DIALOG_MODAL, GTK_MESSAGE_ERROR,
+					 GTK_BUTTONS_CLOSE, NULL);
+
+	gtk_message_dialog_set_markup (GTK_MESSAGE_DIALOG (dialog), span_msg->str);
 
-	gtk_dialog_set_default_response (GTK_DIALOG (dialog), GTK_RESPONSE_OK);
+	gtk_dialog_set_default_response (GTK_DIALOG (dialog), GTK_RESPONSE_CLOSE);
 
```

Comment:
Does this have any implications? I guess not since we do not check the the reutrn of `gtk_dialog_run`..